integration_tests/testUtils.ts

```diff
@@ -0,0 +1,33 @@
+import { Page } from '@playwright/test'
+import tokenVerification from './mockApis/tokenVerification'
+import hmppsAuth, { type UserToken } from './mockApis/hmppsAuth'
+import { resetStubs } from './mockApis/wiremock'
+import bapvUserRoles from '../server/constants/bapvUserRoles'
+import stubComponents from './mockApis/componentApi'
+import TestData from '../server/routes/testutils/testData'
+
+export { resetStubs }
+
+const DEFAULT_ROLES = [`ROLE_${bapvUserRoles.STAFF_USER}`] // TODO sort out ROLE_ prefix
```

Comment:
Bit of a hack to make it work. Need to look into whether the `ROLE_` prefix should be there or not...